cluster/gce/config-default.sh

```diff
@@ -122,7 +122,7 @@ ENABLE_CLUSTER_MONITORING="${KUBE_ENABLE_CLUSTER_MONITORING:-influxdb}"
 # of fluentd running on a node, kubelet need to mark node on which
 # fluentd is not running as a manifest pod with appropriate label.
 # TODO(piosz): remove this in 1.8
-NODE_LABELS="${KUBE_NODE_LABELS:-beta.kubernetes.io/fluentd-ds-ready=true}"
+NODE_LABELS="${KUBE_NODE_LABELS:-beta.kubernetes.io/fluentd-ds-ready=true,beta.kubernetes.io/calico-ds-ready=true}"
```

Comment:
@eparis I was mostly modeling this off the annotation used for the fluentd DaemonSet for consistency. The designation as beta has not had a formal agreement by anyone.  `projectcalico.org/ds-ready=true` sounds fine to me.

@dnardo yes, I think you're right.  Will update.